### DISCUSSION TITLE
indicating if they are sending V2X messages (green) or receiving V2X message (red)

### DISCUSSION THREAD

**Discussion by erouirior** [eclipse/mosaic#103] [Q&A]
Hi, I am facing an error when run my simulation.

![image](https://user-images.githubusercontent.com/10334959/117917572-3f35cf00-b29e-11eb-9db5-d6c4211f7958.png)
It seems like it fails to indicate if they are sending V2X messages or receiving. 

Thanks,
YL

**Comment by DemingCheng** [comment]
I also meet the same question, both the rsu and the vehecles are black. And I can make sure that there are the communication because the rsu can get the messages from the vehicles according to the logs.
I use the topocast to communicate in the cell.
I'm eager for any help, and thank you sincerely in advance.

**Reply by erouirior** [reply]
Hi, my problem is fixed now. You probably can try to modify the RSU and event coordinates, this tool https://www.latlong.net/lat-long-utm.html is very useful.
